### PR TITLE
Add database service documentation

### PR DESCRIPTION
**Pull request by antho1404**

### CODE REVIEW

**Comment by antho1404** [comment]
I did an update of the documentation, the generated content was not so good and inaccurate and also added the same flow like the other services

- definition
- example
- data

File: services/database.md [hunk 1]
```diff
@@ -4,144 +4,351 @@
...
+    taskKey: create
+    inputs:
+      collection: my-collection
+      value:
+        foo: bar
+  - ...
+```
 
-#### Inputs
+#### Data
```
Comment (NicolasMahe) [inline services/database.md:155]:
```suggestion
#### Outputs
```

File: services/database.md [hunk 2]
```diff
@@ -4,144 +4,351 @@
...
+    taskKey: update
+    inputs:
+      collection: my-collection
+      key: XXX
+      value:
+        foo: bar
+  - ...
+```
+
+#### Data
```
Comment (NicolasMahe) [inline services/database.md:204]:
```suggestion
#### Outputs
```

File: services/database.md [hunk 3]
```diff
@@ -4,144 +4,351 @@
...
+      key: XXX
+  - ...
+```
 
-| **Name**       | **Key**      | **Type** | **Description**                        | **Object** |
-| -------------- | ------------ | -------- | -------------------------------------- | ---------- |
-| **collection** | `collection` | `String` | The collection used to store the data. |            |
-| **key**        | `key`        | `String` | The key used to store the data.        |            |
-| **value**      | `value`      | `Object` | The value stored.                      |            |
+#### Data
```
Comment (NicolasMahe) [inline services/database.md:250]:
```suggestion
#### Outputs
```

File: services/database.md [hunk 4]
```diff
@@ -4,144 +4,351 @@
...
-| ---------- | -------- | ---------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **values** | `values` | `Object[]` |                 | [{&quot;key&quot;:&quot;collection&quot;,&quot;description&quot;:&quot;The collection used to store the data.&quot;,&quot;type&quot;:&quot;String&quot;,&quot;object&quot;:[]},{&quot;key&quot;:&quot;key&quot;,&quot;description&quot;:&quot;The key used to store the data.&quot;,&quot;type&quot;:&quot;String&quot;,&quot;object&quot;:[]},{&quot;key&quot;:&quot;value&quot;,&quot;description&quot;:&quot;The value stored.&quot;,&quot;type&quot;:&quot;Object&quot;,&quot;object&quot;:[]}] |
+#### Data
```
Comment (NicolasMahe) [inline services/database.md:295]:
```suggestion
#### Outputs
```

File: services/database.md [hunk 5]
```diff
@@ -4,144 +4,351 @@
...
+      collection: my-collection
+      key: XXX
+  - ...
+```
 
-| **Name**       | **Key**      | **Type** | **Description**                                                           | **Object** |
-| -------------- | ------------ | -------- | ------------------------------------------------------------------------- | ---------- |
-| **collection** | `collection` | `String` | The collection to use to store the data.                                  |            |
-| **key**        | `key`        | `String` | The key to store the data. If not set, an generated UUID v4 will be used. |            |
+#### Data
```
Comment (NicolasMahe) [inline services/database.md:347]:
```suggestion
#### Outputs
```

**Pull request merged**